src/Azure/Shared/Storage/AzureTableDataManager.cs

```diff
@@ -48,23 +48,25 @@ internal class AzureTableDataManager<T> where T : class, ITableEntity, new()
         /// <summary> Connection string for the Azure storage account used to host this table. </summary>
         protected string ConnectionString { get; set; }
 
-        private CloudTable tableReference;
+        public CloudTable Table { get; private set; }
 
-        public CloudTable Table => tableReference;
+        public AzureStoragePolicyOptions StoragePolicyOptions;
```

Comment:
This should not be a public, mutable _field_. At the minimum, it should be a property. It should also be immutable.